DATA_MODEL_CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- **CUMULUS-408**
```

Comment:
Do we need to make dashboard updates for this?

Reply:
Looks like the providers page has fields instead of a JSON form, so yes.

Reply:
It's an optional field for edge case providers that use self-signed certificates, so it's not exactly top priority, but we should file a ticket.

Reply:
Filed CUMULUS-1992 for this.